docs/NIC.md

```diff
@@ -52,16 +57,21 @@ The following command line arguments are supported. Note that providing argument
 ## Included templates
 * **activator_event**: an [event](http://iphonedevwiki.net/index.php/Libactivator#Sending_Events_.28via_LAEvent.29) for Activator.
 * **activator_listener**: a [listener](http://iphonedevwiki.net/index.php/Libactivator#Observing_Events_.28via_LAListener.29) for Activator.
-* **application**: a standard iOS app (for unsandboxed jailbreak use).
+* **application_modern**: a standard iOS app (for unsandboxed jailbreak use).
+* **application_swift**: a standard, swift-based iOS app (for unsandboxed jailbreak use).
+* **control_center_module-11up**: a custom control center module for iOS 11+ deployed via [CCSupport](https://github.com/opa334/CCSupport/wiki).
 * **cydget**: a [Cydget](https://cydia.saurik.com/info/cydget/) lock screen plugin.
 * **flipswitch_switch**: a switch for [Flipswitch](https://github.com/A3Tweaks/Flipswitch).
 * **framework**: a framework to be used by other developers.
 * **library**: a linkable library (e.g. /usr/lib/libblah.dylib).
 * **notification_center_widget**: an iOS 5 – 6 Notification Center Today widget.
-* **notification_center_widget_ios7**: an iOS 7 – 9 Notification Center Today widget.
-* **preference_bundle**: a [PreferenceLoader](http://iphonedevwiki.net/index.php/PreferenceLoader) preference bundle subproject.
+* **notification_center_widget-7up**: an iOS 7 – 9 Notification Center Today widget.
+* **preference_bundle_modern**: a [PreferenceLoader](http://iphonedevwiki.net/index.php/PreferenceLoader) preference bundle subproject.
+* **theme**: a facelift for various UI elements.
```

Comment:
Good call updating this page with all the new templates. For this one I’m wondering if there’s some way we can mention that this is unlike other templates because it’s not building code, it’s just providing the ability to package up a theme using Theos’s packaging functionality.

Reply:
Sure. Updated the description accordingly.